document-generation/generate-document.ts

```diff
@@ -39,7 +41,7 @@ async function baseHandler(event: RequestEvent<GenerateDocumentRequest>): Promis
       return generatePdf(event);
     case DocumentType.INDEPENDENT_GOVERNMENT_COST_ESTIMATE:
       return generateXlsx(event);
-    case DocumentType.INCREMENTAL_FUNDING_PLAN:
+    case DocumentType.INCREMENTAL_FUNDING_PLAN || DocumentType.EVALUATION_PLAN:
```

Comment:
Control should flow through the switch/case statement until it hits a break or return statement.
I would expect something like this rather than introducing a logical or in the case value :
```suggestion
    case DocumentType.INCREMENTAL_FUNDING_PLAN:
    case DocumentType.EVALUATION_PLAN:
    // return generateWordDocument(event); -- satisfies both of the above
```